### ISSUE TITLE
UDMSE v3.0.13

### ISSUE THREAD

**Issue by Navish360** [pbrah/wpa_supplicant-udmpro#21]
I've had wpa_supplicant working for a few years now on 1.x , and 2.x but with the new version upgrade to the UDMSE seems like it broke after the update.  I've done a full reset on the UDMSE, and reinstalled Podman, but keep getting an error when trying to run the podman container. When running `podman ps -a` I see the container is sitting in a "Created' status. I attempted to run the container using `podman start <id>` but getting the message below. 

```
root@KenaxanetworksUDM-SE:~# podman start 87a338a89f32
ERRO[0000] json-file logging specified but not supported. Choosing k8s-file logging instead
Error: OCI runtime error: unable to start container "87a338a89f3247c970b160e1a2b9a9f93fdb7dfc92914d0bb745a7d4cb801653":
runc create failed: unable to start container process: error during container init: error setting cgroup config for procHooks process: 
bpf_prog_query(BPF_CGROUP_DEVICE) failed: function not implemented
```

I have no idea what could be causing it or what dependency I'm missing to get the container started.    

-Navish A.

**Comment by ethansawicki** [comment]
> I've had wpa_supplicant working for a few years now on 1.x , and 2.x but with the new version upgrade to the UDMSE seems like it broke after the update. I've done a full reset on the UDMSE, and reinstalled Podman, but keep getting an error when trying to run the podman container. When running `podman ps -a` I see the container is sitting in a "Created' status. I attempted to run the container using `podman start <id>` but getting the message below.
> 
> ```
> root@KenaxanetworksUDM-SE:~# podman start 87a338a89f32
> ERRO[0000] json-file logging specified but not supported. Choosing k8s-file logging instead
> Error: OCI runtime error: unable to start container "87a338a89f3247c970b160e1a2b9a9f93fdb7dfc92914d0bb745a7d4cb801653":
> runc create failed: unable to start container process: error during container init: error setting cgroup config for procHooks process: 
> bpf_prog_query(BPF_CGROUP_DEVICE) failed: function not implemented
> ```
> 
> I have no idea what could be causing it or what dependency I'm missing to get the container started.
> 
> -Navish A.

The UDMSE doesn't have podman installed. They are removing it on the UDMP/UXG when 2.0 comes to it. There is still a way to bypass without podman.

**Comment by Navish360** [comment]
> > I've had wpa_supplicant working for a few years now on 1.x , and 2.x but with the new version upgrade to the UDMSE seems like it broke after the update. I've done a full reset on the UDMSE, and reinstalled Podman, but keep getting an error when trying to run the podman container. When running `podman ps -a` I see the container is sitting in a "Created' status. I attempted to run the container using `podman start <id>` but getting the message below.
> > ```
> > root@KenaxanetworksUDM-SE:~# podman start 87a338a89f32
> > ERRO[0000] json-file logging specified but not supported. Choosing k8s-file logging instead
> > Error: OCI runtime error: unable to start container "87a338a89f3247c970b160e1a2b9a9f93fdb7dfc92914d0bb745a7d4cb801653":
> > runc create failed: unable to start container process: error during container init: error setting cgroup config for procHooks process: 
> > bpf_prog_query(BPF_CGROUP_DEVICE) failed: function not implemented
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I have no idea what could be causing it or what dependency I'm missing to get the container started.
> > -Navish A.
> 
> The UDMSE doesn't have podman installed. They are removing it on the UDMP/UXG when 2.0 comes to it. There is still a way to bypass without podman.

Yeah I know they removed podman out of the UDMSE awhile back, but it's still possible to install using unifi-os utilities by @boostchicken. If 3.x finally breaks integration on Podman I can't think of what other ways it would be possible to bypass the ATT gateway than wpa_supplicant.

**Comment by ethansawicki** [comment]
I will say there have been other developments on the bypass space with buying your own ONT. Theres a big discussion about it on dslreports.com forums. Look for "bye bye 802.1x" in the att uverse forum section. But look into this post on reddit [https://old.reddit.com/r/Ubiquiti/comments/q7qk3m/udm_se_persisted_debian_packages_installation/](url) Look in the comments it should help

**Comment by Navish360** [comment]
Yep, I've been after that bypass but unfortunately I'm one of the few that still require wpa_supplicant to be ran alongside the SFP ONT bypass.  It's still unknown why some people require wpa_supplicant, and some don't when doing that method.

**Comment by ethansawicki** [comment]
Yeah ive seen that but look into that reddit thread some asked about doing the bypass and there is decent instructions in a comment thread.

**Comment by Navish360** [comment]
I'll take a look into
